meson.build

```diff
@@ -1,6 +1,6 @@
 project('p11-kit', 'c',
         version: '0.25.3',
-        meson_version: '>= 0.51')
+        meson_version: '>= 0.63')
```

Comment:
CodeQL CI is failing because they only have 0.61.2 installed:
```console
meson.build:1:0: ERROR: Meson version is 0.61.2 but project requires >= 0.63
```
I guess we could stick with `>= 0.51`.

Reply:
I have removed the last commit bumping the meson version requirement. Recent meson ssems to be smart enough to not throw
> meson.build:67: WARNING: Project targets '>= 0.51' but uses feature introduced in '0.63.0': add_project_dependencies.
for inactive code, i.e. the warning is only shown on hosts matching
```
if ['gnu', 'gnu/kfreebsd'].contains(host_system)
```